### ISSUE TITLE
Alert extensions

### ISSUE THREAD

**Issue by edvin** [edvin/tornadofx#68]
Provide convenience function for working with Alerts.

**Comment by thomasnield** [comment]
Are we talking about those Dialog factories that were implemented into JavaFX 8 well after its release?

**Comment by edvin** [comment]
That's right. Their API is quite OK already, but we could still streamline a little :)

**Comment by thomasnield** [comment]
I agree. I'll see I I can help with this and a few of the other issues.

**Comment by edvin** [comment]
Cool! I'm in Oslo from today until Sunday, but hopefully I'll be able to put in a few hours every night atleast.

**Comment by edvin** [comment]
Implemented and will be in TornadoFX 1.4.3.

````kotlin
alert(CONFIRMATION, "Confirm delete", "Do you want to delete ${customer.name}?") {
    if (it == ButtonType.OK)
        customerController.deleteCustomer(customer)
}
```

Multiple outcomes:

```kotlin
alert(CONFIRMATION, "Choose action", "What do you want to do?", APPLY, NEXT, CLOSE) {
    when (it) {
        APPLY -> applyIt()
        NEXT -> gotoNext()
        CLOSE -> closeWindow()
    }
}
```
